hw-2/server.ts

```diff
@@ -0,0 +1,64 @@
+import express = require('express');
+import {UsersStorage} from './storage/users';
+import bodyParser = require('body-parser');
+import {ContainerTypes,
+        // Use this as a replacement for express.Request
```

Comment:
there should be no comments in import statement - if you want to comment methods - you should comment them where they are defined